server/channels/app/platform/cluster.go

```diff
@@ -181,7 +181,7 @@ func (ps *PlatformService) InvokeClusterLeaderChangedListeners() {
 
 func (ps *PlatformService) Publish(message *model.WebSocketEvent) {
 	if ps.metricsIFace != nil {
-		ps.metricsIFace.IncrementWebsocketEvent(message.EventType())
+		ps.metricsIFace.IncrementWebsocketEvent(string(message.EventType()))
```

Comment:
Urg, it's a bummer we can't change the API here as it's closed-source enterprise code. Hence, this is fine for now.